src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -616,6 +588,36 @@ export class DashboardAppController {
       }
     );
 
+    const getNavBarProps = () => {
+      const isFullScreenMode = dashboardStateManager.getFullScreenMode();
+      const screenTitle = dashboardStateManager.getTitle();
+      return {
+        appName: 'dashboard',
+        config: $scope.isVisible ? $scope.topNavMenu : undefined,
+        className: isFullScreenMode ? 'kbnTopNavMenu-isFullScreen' : undefined,
```

Comment:
//nit 
I think it would be simpler if the class would be `fullscreen`
It's anyway scoped in SCSS and it might be useful elsewhere.

Reply:
I'm not sure about this one, I think the convention is we prefix the classname with the component name.
cc @cchaos